client/integration-tests/serializable-model/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+quarkus.openapi-generator.codegen.spec.quarkus_serializable_openapi_yaml.serializable-model=true
+quarkus.openapi-generator.codegen.spec.quarkus_serializable_openapi_yaml.base-package=io.quarkiverse.serializable
+quarkus.openapi-generator.codegen.spec.quarkus_non_serializable_openapi_yaml.base-package=io.quarkiverse.non.serializable
```

Comment:
Please use `org.acme` instead. `io.quarkiverse` can lead to confusion.

Reply:
Done.